topic-operator/src/main/java/io/strimzi/operator/topic/KafkaStreamsTopicStoreService.java

```diff
@@ -110,8 +111,18 @@ private TopicStore createKafkaTopicStore(Config config, Properties kafkaProperti
             }
         };
 
-        Topology topology = new TopicStoreTopologyProvider(storeTopic, storeName, kafkaProperties, dispatcher).get();
-        streams = new KafkaStreams(topology, kafkaProperties);
+        // provide some Kafka Streams defaults
+        Properties streamsProperties = new Properties();
+        streamsProperties.putAll(kafkaProperties); // don't add them as defaults?
```

Comment:
Do we really need the `// don't add them as defaults?` comment? It do not think it adds any value.